Dockerfile.daemon

```diff
@@ -18,8 +18,10 @@ ENV HOME=/home/necoperf
 COPY --from=flatcar /bin/ /bin/
 COPY --from=flatcar /sbin/setcap /sbin/setcap
 COPY --from=flatcar /lib64 /lib64
+COPY --from=flatcar /usr/lib64 /usr/lib64
 COPY --from=flatcar /usr/bin/perf /usr/bin/sleep /usr/bin/
 
+
```

Comment:
Please delete the unnecessary line breaks.

Reply:
deleted the empty line